platforms/hyperledger-fabric/charts/osn_create_channel/templates/osn_create_channel.yaml

```diff
@@ -0,0 +1,190 @@
+            TLSCACERTS=$(echo ${VAULT_SECRET} | jq -r '.["tlscacerts"]')
+
+            OUTPUT_PATH="${MOUNT_PATH}/${ORDERER_NAME}/msp"
+            mkdir -p ${OUTPUT_PATH}/admincerts
+            mkdir -p ${OUTPUT_PATH}/cacerts
+            mkdir -p ${OUTPUT_PATH}/keystore
+            mkdir -p ${OUTPUT_PATH}/signcerts
+            mkdir -p ${OUTPUT_PATH}/tlscacerts
+
+            echo "${ADMINCERT}" >> ${OUTPUT_PATH}/admincerts/admin.crt
+            echo "${CACERTS}" >> ${OUTPUT_PATH}/cacerts/ca.crt
+            echo "${KEYSTORE}" >> ${OUTPUT_PATH}/keystore/server.key
+            echo "${SIGNCERTS}" >> ${OUTPUT_PATH}/signcerts/server.crt
+            echo "${TLSCACERTS}" >> ${OUTPUT_PATH}/tlscacerts/tlsca.crt
+
+            # Check if orderers tls already created
+            vaultBevelFunc "readJson" "${VAULT_ORDERER_SECRET_PREFIX}/${ORDERER_NAME}.${COMPONENT_NAME}/tls" 
+
+            CACERT=$(echo ${VAULT_SECRET} | jq -r '.["ca.crt"]')
+            CLIENTSEVER=$(echo ${VAULT_SECRET} | jq -r '.["server.crt"]')
+            CLIENTKEY=$(echo ${VAULT_SECRET} | jq -r '.["server.key"]')
+
+            OUTPUT_PATH="${MOUNT_PATH}/${ORDERER_NAME}/tls"
+            mkdir -p ${OUTPUT_PATH}
+
+            echo "${CACERT}" >> ${OUTPUT_PATH}/ca.crt
+            echo "${CLIENTSEVER}" >> ${OUTPUT_PATH}/server.crt
+            echo "${CLIENTKEY}" >> ${OUTPUT_PATH}/server.key
+          done
+
+        volumeMounts:
+        {{ if .Values.vault.tls  }}
+        - name: vaultca
+          mountPath: "/etc/ssl/certs/"
+          readOnly: true
+        {{ end }}
+        - name: certificates
+          mountPath: /secret
+        - name: scripts-volume
+          mountPath: /scripts/bevel-vault.sh
+          subPath: bevel-vault.sh
+      containers:
+      - name: createchannel
+        image: {{ $.Values.metadata.images.fabrictools }}  
+        imagePullPolicy: IfNotPresent
+        stdin: true
+        tty: true
+        env:
+        - name: ORDERERS_INFO
+          value: "{{ $.Values.orderers.orderer_info }}"
+        - name: NAMESPACE
+          value: "{{ $.Values.metadata.namespace }}"
+        command: ["sh", "-c"]
+        args:
+        - |-
+          #!/usr/bin/env sh
+          echo "Fetch genesis file..."
+          cat ./genesis/genesis.block.base64 | base64 -d > orderer.genesis.block
+
+          list=$(echo "$ORDERERS_INFO" | tr "*" "\n")
+          for ORDERER_NAME in $list
+          do
+            ADMIN_TLS_PRIVATE_KEY="${ADMIN_TLS_CERTS}/${ORDERER_NAME}/tls/server.key"
+            ADMIN_TLS_SIGN_CERT="${ADMIN_TLS_CERTS}/${ORDERER_NAME}/tls/server.crt"
+            OSN_TLS_CA_ROOT_CERT="${ADMIN_TLS_CERTS}/${ORDERER_NAME}/tls/ca.crt"
+            ORDERER_URL="${ORDERER_NAME}.${NAMESPACE}:7055"
+
+            RESPONSE=$(osnadmin channel list --channelID ${CHANNEL_NAME} -o "${ORDERER_URL}" --ca-file "${OSN_TLS_CA_ROOT_CERT}" --client-cert "${ADMIN_TLS_SIGN_CERT}" --client-key "${ADMIN_TLS_PRIVATE_KEY}")
```

Comment:
Can you rename this variable to `CHANNEL_LIST_QUERY_RESPONSE`

Reply:
Done